src/app/Components/ArtworkGrids/InfiniteScrollArtworksGrid.tsx

```diff
@@ -339,7 +338,7 @@ const InfiniteScrollArtworksGrid: React.FC<Props & PrivateProps> = ({
         const displayToolTip =
           enableMyCollectionSellOnboarding &&
           isMyCollection &&
-          !isDismissed(PROGRESSIVE_ONBOARDING_MY_COLLECTION_SELL_THIS_WORK).status &&
+          !isDismissed("my-collection-sell-this-work").status &&
```

Comment:
Shouldn't we use the constants instead of the strings everywhere? 

```suggestion
          !isDismissed(PROGRESSIVE_ONBOARDING_MY_COLLECTION_SELL_THIS_WORK).status &&
```

Reply:
As Carlos mentioned [here](https://github.com/artsy/eigen/pull/11404#discussion_r1918585174) it's okay either way because it can be auto-completed by IDEs. I prefer a string as it is more readable imo